specification/dartLangSpec.tex

```diff
@@ -15785,7 +15782,7 @@ \subsection{Yield-Each}
 It is a compile-time error if $T_e$ may not be assigned to $T_f$.
```

Comment:
Lower case "i" at the beginning? Maybe comma before it.

Reply:
Maybe this is a github UI glitch? The line `It is a compile-time error if $T_e$ may not be assigned to $T_f$.` occurs just after a line containing commentary that ends in `.`.

Reply:
True, there are lines missing from the view which cannot be expanded.